extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -714,7 +734,11 @@ export class AmpA4A extends AMP.BaseElement {
         layoutAdPromiseDelay: Math.round(delta),
         isAmpCreative: !!creativeMetaData,
       });
-      if (creativeMetaData) {
+      if (!creativeMetaData) {
+        // Non-AMP creative case, will verify ad url existence.
+        return this.renderNonAmpCreative_();
+      }
+      if (!creativeMetaData.collapse) {
```

Comment:
If it were me, I would have made this:

`if (creativeMetaData.collapse) {
   return Promise.resolve();
}
// continue you renderAmpCreative flow...`